### ISSUE TITLE
Only Facebook Sign Up ?

### ISSUE THREAD

**Issue by Sammeeey** [slyapustin/django-classified-demo#122]
Is it only possible to sign up with a facebook account and not via email?

**Comment by slyapustin** [comment]
@Sammeeey Yes, it has a full integration with Django Social Auth. Please check their documentation on how to extend authorization: https://python-social-auth.readthedocs.io/en/latest/configuration/django.html
